packs/kingmaker-bestiary/skeletal-tiger-lord.json

```diff
@@ -501,7 +501,7 @@
                 "category": "defensive",
                 "deathNote": true,
                 "description": {
-                    "value": "<p>When a skeletal Tiger Lord is destroyed, its bones clatter to the ground in a tangle and its armor and weapons crumble to dust. It automatically reforms with its armor and weapons intact (regaining a full quiver of 20 arrows) 1 hour later at 1 Hit Point, allowing its fast healing to resume healing it thereafter. If the trap in area D5 is disabled, the skeleton loses this ability; otherwise, the only way to permanently destroy a skeletal Tiger Lord is to reduce it to 0 Hit Points with vitality damage or evil damage.</p>"
+                    "value": "<p>When a skeletal Tiger Lord is destroyed, its bones clatter to the ground in a tangle and its armor and weapons crumble to dust. It automatically reforms with its armor and weapons intact (regaining a full quiver of 20 arrows) 1 hour later at 1 Hit Point, allowing its fast healing to resume healing it thereafter. If the trap in area D5 is disabled, the skeleton loses this ability; otherwise, the only way to permanently destroy a skeletal Tiger Lord is to reduce it to 0 Hit Points with vitality damage or holy damage.</p>"
```

Comment:
I'm unsure if this should be holy or spirit damage.

Reply:
Oh
That's a typo - I checked the PDF for Kingmaker and this was entered incorrectly as Evil damage.

Reply:
Good catch. I would guess that your correction to holy best fits the design intent?

Reply:
Yeah - the way I interpret it is if you are sanctified holy and deal damage it stops the regeneration.